buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -21,7 +21,7 @@ object Dependencies {
     const val androidx_multidex = "androidx.multidex:multidex:2.0.1"
     const val androidx_preference_ktx = "androidx.preference:preference-ktx:1.2.0"
     const val androidx_fragment_ktx = "androidx.fragment:fragment-ktx:${Versions.androidx_fragment}"
-    const val android_material = "com.google.android.material:material:1.8.0"
+    const val android_material = "com.google.android.material:material:1.7.0"
```

Comment:
Looks like there are problems using `MaterialCardView` with a Material 3 style and a non-Material 3 theme that we need to solve to upgrade to 1.8.0.